app/services/mixins/group_validate_mixin.rb

```diff
@@ -24,6 +24,12 @@ def runtime_validate_group(request)
       group = ensure_group(request.group_ref, request.context)
     rescue Exceptions::UserError => e
       error_action(request, e.message)
+    rescue RBACApiClient::ApiError => e
+      error_action(request, e.message)
```

Comment:
Do not send `e.message` to action because it is meant to be shown to user. 
Consider the following for all Non UserError
```
  error_action(request, "Internal error while validating a group. Check log for details.")
  Rails.logger.error(["Caught an error while validating a group before forwarding the request to PAM.", e.message, *e.backtrace].join($RS))
```

Reply:
I think you meant `it is meant to be shown to user` to be `it is *not* meant to be shown to user`, and I agree 👍

Reply:
I got your means 😄